b2share/modules/main/restful.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from flask.ext.restful import Resource
+
+# ----------------------------------------------- COMMUNITIES
+
+
+class CommunityList(Resource):
+    def get(self, **kwargs):
+        """
+        Retrieve list of communities.
+        """
+        return None
+
+
+class CommunityRes(Resource):
+    def post(self, **kwargs):
+        """
+        Creates a new community that has associated a new metadata fieldset.
+        Only administrators can use it.
+        parameter: name, description, logo
+        """
+        return None
+
+
+class Community(Resource):
+    def get(self, community_id, **kwargs):
+        """
+        Get a community metadata and description.
+        """
+        return None
+
+    def patch(self, community_id, **kwargs):
+        """
+        Modify a community
+        """
+        return None
+
+
+# ----------------------------------------------- SCHEMA
+
+
+class schemaList(Resource):
```

Comment:
Let's agree to start all class names with Capital Letters: https://www.python.org/dev/peps/pep-0008/#class-names